airflow/cli/commands/connection_command.py

```diff
@@ -28,9 +28,9 @@
 from airflow.exceptions import AirflowNotFoundException
 from airflow.hooks.base import BaseHook
 from airflow.models import Connection
-from airflow.secrets.local_filesystem import _create_connection, load_connections_dict
 from airflow.utils import cli as cli_utils, yaml
 from airflow.utils.cli import suppress_logs_and_warning
+from airflow.utils.parse import parse_file
```

Comment:
Hmmmm, I'm not really happy with this module name -- it doesn't _really_ tell me what its doing, not in specific enough terms.

We've already got `airflow.utils.file` that we could include this in -- I think I prefer that over a new file.

"parse_file" is also a bit ambigious, but I can't think of a better name.

Reply:
Got rid of utils/parse.py and left the functionality where it was in `secrets/local_filesystem.py`.